### ISSUE TITLE
Trigger View Handler should return event object

### ISSUE THREAD

**Issue by frankcortes** [marionettejs/backbone.marionette#2975]
Hello everyone, I was reading [Marionette's docs about Trigger Handler arguments](http://marionettejs.com/docs/v2.4.4/marionette.view.html#trigger-handler-arguments) and it's weird for me the fact that I cannot retrieve the original `event` object. 
Now, if I want get the original event and execute a trigger I have to do this:

``` javascript
events: { 'click .my-class': 'onAwesomeClick' },
//...some code
onAwesomeClick: function(event) {
   this.trigger('real:awesome:click', event);
},
onRealAwesomeClick: function(event) {
   //Do everything you want
}
```

This looks like me like it's a kind of proxy, because if I could do this

``` javascript
triggers: { 'click .my-class': 'real:awesome:click'  },
//...some code
onRealAwesomeClick: function(args) { //where args.event contains the original event
   //Do everything you want
},
```

It would rather use it better, and it would remove a lot of boilerplate. What are your thoughts about this?

**Comment by paulfalgout** [comment]
This is a dupe of https://github.com/marionettejs/backbone.marionette/pull/2690

**Comment by paulfalgout** [comment]
We might consider this in v3 as a 2nd argument for `triggers`? The only caveat here is that we should document it so that we strongly recommend against using the event outside of the event's own view.  Using the `onTriggeredMethod` calls inside a View is a pretty regular pattern and it is awkward to _have_ to use events on the View if the original event is needed.

But now that there's no longer an odd trigger argument object, I'm ok with it.  I'm not pushing for it, but I'm not against it either.

Thoughts? @marionettejs/marionette-core

**Comment by JSteunou** [comment]
Solved by #3261
